packages/switch/NativeControl.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export default class NativeControl extends React.Component {
+  render() {
+    const {
+      setRippleActivator,
+      ...otherProps
+    } = this.props;
+
+    return (
+      <input
+        type='checkbox'
+        role='switch'
+        className='mdc-switch__native-control'
+        ref={setRippleActivator}
+        {...otherProps}
+      />
+    );
+  }
+}
+
+NativeControl.propTypes = {
+  checked: PropTypes.bool,
+  disabled: PropTypes.bool,
+  id: PropTypes.string,
+  onChange: PropTypes.func,
+  setRippleActivator: PropTypes.oneOfType([
```

Comment:
can this just be `rippleActivator`, which I assume is an Element, since it gets passed to a ref

Reply:
It's a `ref` which gets this PropType definition according to [StackOverflow](https://stackoverflow.com/questions/48007326/what-is-the-correct-proptype-for-a-ref). Changed the name to `rippleActivatorRef` to make it clearer.